### PR TITLE
Add Exception Manager

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We need an exception manager to help us view and skip unprocessable messages with zero downtime.

# What has changed
Added the Exception Manager.

# How to test?
Build. Run. Publish bad message. Check the logs. Check the Exception Manager.

# Links
Trello: https://trello.com/c/2rkYNAMa

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[Exception Manager v1.0](https://trello.com/c/2rkYNAMa/1266-exception-manager-v10)

**Comment by NickGrantONS** [comment]
[Exception Manager v1.0](https://trello.com/c/2rkYNAMa/1266-exception-manager-v10)

File: .codecov.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ coverage:
   status:
     project:
       default:
-        target: 75%
+        target: 50%
```
Comment (AdamHawtin) [inline .codecov.yml]: 👀

Comment (NickGrantONS) [reply .codecov.yml]: can put this back up now... it's at 78% coverage

**Pull request merged**